sources/packages/web/src/assets/css/global-style-variables.scss

```diff
@@ -41,6 +41,8 @@ $line-height-header-sm: 22px;
 $formio-content: 16px;
 $formio-content-lh: 24px;
 $formio-header-margin: 0.5rem;
+$formio-header-wt: 700;
+$formio-category-header-wt: 800;
```

Comment:
Is it really needed? It it affecting the final UI?

Reply:
I see why @henryoforeh-aot used weight 800. Figma used 800 as weight for bold in the design system. 
![image](https://user-images.githubusercontent.com/54600590/154128637-4bcde7d8-d9c1-4ab6-ad24-f6ba4e6330b7.png)
 @henryoforeh-aot Can you please check with Lynn to see if we use font-weight 800 for bold and not 700.

Reply:
@dheepak-aot I believe that the BCSans used in the project does not even support 800, that is why I asked if setting 800 was causing some impact. If BC Sans does not support it and if it is not causing an impact in the UI we will not be using it, right?

Reply:
I have checked with Lynn, we will not use 800 for font-weight. Thanks for the clarification.

Reply:
@andrewsignori-aot Yes. I didn't have a look at BCSans while making the comment. @henryoforeh-aot thanks for checking with Lynn.